syntax/jsonc.vim

```diff
@@ -20,7 +20,9 @@ endif
 runtime syntax/json.vim
 
 " Remove syntax group for comments treated as errors
-syn clear jsonCommentError
+if !exists("g:vim_json_warnings") && g:vim_json_warnings
```

Comment:
So, this branch is taken only when `g:vim_json_warnings` does not exist and is simultaneously true?  Seems like the branch would either not be taken (if `g:vim_json_warnings` exists) or cause an error (if it does not exist).

Reply:
Yup, dumb mistake, that should be `||`.